pkgs/development/python-modules/jsonfield/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchPypi, buildPythonPackage, django, pytest, pytest-django }:
+
+buildPythonPackage rec {
+  pname = "jsonfield";
+  version = "3.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yl828cd0m8jsyr4di6hcjdqmi31ijh5vk57mbpfl7p2gmcq8kky";
+  };
+
+  checkInputs = [ pytest pytest-django ];
+
+  checkPhase = ''
+    PYTHONPATH="$(pwd):$PYTHONPATH" \
+    DJANGO_SETTINGS_MODULE=tests.settings \
+    pytest #tests/tests.py
```

Comment:
We can't use pytestCheckHook here, right?

Reply:
I haven't figured out how, but I guess I'll better just not customize this phase.